Containerfile

```diff
@@ -1,6 +1,11 @@
 ARG TAG=latest
 
 FROM cfaprdbatchcr.azurecr.io/pyrenew-hew-dependencies:${TAG}
+ARG GIT_COMMIT_SHA
```

Comment:
should this and the branch name be arguments to the container build call or should the container build engine itself invoke `git` to determine it? The latter seems more robust: will also work with logal builds, will reduce GH actions magic, and it's a pretty minimal dep for container building.